tapdb/sqlc/queries/assets.sql

```diff
@@ -1016,3 +1017,22 @@ FROM genesis_assets assets
 JOIN assets_meta
     ON assets.meta_data_id = assets_meta.meta_id
 WHERE assets.asset_id = $1;
+
+-- Upsert a record into the mint_anchor_uni_commitments table.
+-- If a record with the same batch_id and group_key already exists, update the
+-- existing record. Otherwise, insert a new record.
+-- name: UpsertMintAnchorUniCommitment :one
+INSERT INTO mint_anchor_uni_commitments (
+    id, batch_id, tx_output_index, taproot_internal_key, group_key
+)
+VALUES ($1, $2, $3, $4, $5)
+ON CONFLICT(batch_id, tx_output_index) DO UPDATE SET
+    tx_output_index = EXCLUDED.group_key,
+    taproot_internal_key = EXCLUDED.taproot_internal_key
+RETURNING id;
```

Comment:
I still think we should cover this with a unit test. But can be done in a follow-up PR.